# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mc/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

--- mc-4.1.40-pre8/src/cons.handler.c.orig	2004-05-06 00:02:05.414546544 +0200
+++ mc-4.1.40-pre8/src/cons.handler.c	2004-05-06 00:02:36.968749576 +0200
@@ -205,6 +205,7 @@
 	break;
     default:
 	/* Nothing */
+	;
     }
 }
 
